### DISCUSSION TITLE
Interruttori 503

### DISCUSSION THREAD

**Discussion by Supertux02** [indomus/forum#308] [HUB: Home Assistant]
Buongiorno,
arrivo subito al punto saltando i preamboli. Sto ragionando su come ristrutturare casa e, per adesso, sono arrivato a pensare agli interruttori elettrici per le luci (quelli canonici da scatola 503). Vorrei qualcosa di "normale" (quindi non roba quadrata o alla stregua di pulsanti zigbee esterni o che vadano a batteria), che funzionino in "locale" (voglio comunque poter accendere la luce anche se non ho internet)e che si integrino bene in home assistant (e poter fare eventuali automazioni, scene e via discorrendo). 
Avevo pensato anche a qualcosa di flashabile con tasmota (non l'ho mai fatto su un interruttore, ma mi organizzerei). Il problema è che non ho trovato nulla di papabile. L'unica cosa che si avvicina è il bticino living now ma, a parte che non mi piacciono per niente, l'integrazione con home assitant non mi da sufficienti garanzie di funzionamento nel tempo. Avete qualche idea su prodotti affidabili, dal costo giusto ed esteticamente "normali"?
Avevo pensato anche a dei pulsanti da abbinare a degli shelly 1 ma credo che nella 503 poi sono costretto a fare un grande appiccicaticcio se ci volessi mettere anche solo 2 interruttori. 
Grazie :)

**Comment by essegi65** [comment]
Esperienza mia personale ho scelto proprio living now di bticino (tra l'altro adesso esiste anche la serie living smart) e ho collegato a HA il tutto con l'integrazione homekit, proprio per non perdere anche la possibilità di gestirla con l'app proprietaria (inoltre al momento non ho nessun componente zigbee). Devo dire che non trovo difficoltà a gestirla con HA sia a livello di prese e interruttori singoli (tranne ovviamente per le prese la lettura del consumo) sia attraverso automazioni. Inoltre tutto quanto non avuto problemi a collegarlo anche a google homeassistant.
Se vuoi altre informazioni chiedi pure

**Reply by snakuzzo** [reply]
@essegi65 parli di `Homekit Controller` non `Homekit` ... specifico perché sono due cose completamente diverse

**Reply by snakuzzo** [reply]
Ad ogni modo @Supertux02 se non ami l'idea di utilizzare attuatori come Shelly o Sonoff, ti consiglierei di usare qualcosa tipo Vimar o appunto Bticino. Se ne fai un discorso di estetica ... non saprei cosa dirti. Se parliamo di funzionalità hai due strade. Una è quella che suggeriva @essegi65 lasciando il loro gateway ed utilizzando [Homekit Controller](https://www.home-assistant.io/integrations/homekit_controller/), l'altra è usare un tuo coordinator Zigbee, magari basato su chipset Texas Instruments come [Z-Bee Duo](https://indomus.it/componenti/z-bee-duo-zigbee-coordinator-usb-hat-per-raspberry-pi-zero/). Qui c'è la lista dei [device supportati](https://www.zigbee2mqtt.io/information/supported_devices.html#bticino) da Zigbee2mqtt appunto. Tieni presente che, anche se i device non sono supportati, con Zigbee2mqtt c'è una [procedura](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html) ad hoc per far aggiungere e supportare nuovi device

**Reply by Supertux02** [reply]
Forse mi sono spiegato male. Adoro shelly e sonoff (più shelly, ma lasciamo perdere) ma credo che dovendo fare scatole 503 mi venga tutto compresso. A quel punto dovrei mettere 2 interruttori per scatola (dato che il terzo slot sarebbe occupato da uno shelly 2.5), quindi penso che dovrei mettere più scatole per avere lo stesso risultato. Per questo motivo avrei preferito una soluzione con smart switch (per capirsi sulla tipologia di meross o iotty) che però o sono dipendenti dalla rete internet (meross) o non sono compatibili con home assistant (iotty). In ogni caso preferirei qualcosa a pulsanti fisici e non a sfioramento, ma non vorrei mettere troppi paletti.
Il mio "problema" con la linea living now di bticino è che proprio non mi piace...con il resto della casa è un pugno in un occhio -.-' Quindi preferirei un'altro tipo di soluzione...anche se mi sembra di capire che non ce ne siano molte.

**Reply by essegi65** [reply]
Prova a vedere la linea living smart sempre bticino, è differente come estetica dalla linea living now.

**Reply by snakuzzo** [reply]
@Supertux02 comunque avevo capito benissimo e ti eri spiegato bene :)
Il fatto è che le soluzioni quelle sono (per ora). Con le 503 secondo me più di uno shelly non riesci a mettere...verrebbe tutto troppo schiacciato. Al massimo potresti sfruttare spazio in profondità rompendo la scatola, ma "personalmente" non mi piace come cosa.
Con uno shelly 2.5  risolvi se hai intenzione di usare massimo 2 interruttori/pulsanti. Tra l'altro entra perfettamente nello spazio di un tappo (almeno con la serie economica che ho io).

**Reply by Supertux02** [reply]
@essegi65 In effetti è diversa...però anche questa non mi convince più di tanto. La leva è molto a filo con gli altri tasti e con la cornice e risulta scomoda (in casa ho una linea non smart bticino che è esteticamente uguale a quella, presi quando rifacemmo il bagno...si prese perché ci fidavamo del brand ed in effetti funziona bene...ma è scomodissimo trovare l'interruttore al buio...spesso mi trovo a spingere sulla cornice convinto che sia l'interruttore)
@snakuzzo temo che alla fine debba ripiegare sugli shelly 2.5 su tutti gli interruttori e sulle prese di casa. Che tu sappia, quanta potenza possono reggere gli shelly 2.5? Perché se su un interruttore il problema non si pone, non è altrettanto vero su una presa...ci posso attaccare un phon o una caldaia elettrica? Io credo che regga fino ai 16A per linea (quindi 32A totali)...ma non trovo quanti W possa reggere...
Grazie :)

**Reply by essegi65** [reply]
Su le prese ti consiglio di montare shelly 1pm che regge il carico fino 3 kw. e ti conteggia anche i relativi consumi (visibili anche in HA con MQTT).
p.s. trovare un interruttore al buio oggi non è un problema per nessuna azienda che li produce, ci puoi inserire delle spie a led che li fanno vedere al buio o si accendono quando accendi la luce della stanza

**Reply by xyzmauro** [reply]
> @essegi65 In effetti è diversa...però anche questa non mi convince più di tanto. La leva è molto a filo con gli altri tasti e con la cornice e risulta scomoda (in casa ho una linea non smart bticino che è esteticamente uguale a quella, presi quando rifacemmo il bagno...si prese perché ci fidavamo del brand ed in effetti funziona bene...ma è scomodissimo trovare l'interruttore al buio...spesso mi trovo a spingere sulla cornice convinto che sia l'interruttore)
> @snakuzzo temo che alla fine debba ripiegare sugli shelly 2.5 su tutti gli interruttori e sulle prese di casa. Che tu sappia, quanta potenza possono reggere gli shelly 2.5? Perché se su un interruttore il problema non si pone, non è altrettanto vero su una presa...ci posso attaccare un phon o una caldaia elettrica? Io credo che regga fino ai 16A per linea (quindi 32A totali)...ma non trovo quanti W possa reggere...
> Grazie :)

non so se per Living Smart intendi la Livinglight, se si i led sono integrati e si possono accendere sempre, mai oppure quando la luce è accesa, basta pubblicare sul topic quello che vuoi...ad esempio con un'automazione a una certa ora della sera li accendi e alla notte li becchi per forza, all'alba li spegni (solo quelli che ti interessano chiaramente)

shelly 2.5 sono 10A quindi 2200W

**Reply by Larry-78** [reply]
Ciao @Supertux02 visto che ti trovi nella condizione di ristrutturare perché invece di pensare di sistemare i dispositivi nelle 503 non pensi di creare cassette di derivazione più grandi? Anch'io ho ristrutturato da poco e ho progettato il tutto sulla base di un impianto normale utilizzando relè per i punti derivati e interruttori per i punti singoli, tutto della serie living light (ovviamente si potrebbe utilizzare qualsiasi serie in commercio) sistemando i dispositivi nelle cassette di derivazione. Ho utilizzato sonoff mini per punti luce e sonoff dual per le tapparelle, mi trovo benissimo. Entrando un pò nello specifico ho utilizzato relè Finder 26 i quali permettono la gestione separata dei contatti e quindi facilmente collegabili al mini. Lo schema rimane quello di un impianto standard a relè dove il pulsante agisce sulla bobina ma il contatto agisce sul mini ai morsetti S1 S2, in fine l'uscita del mini andrà alle lampade. Il vantaggio sta nella praticità di gestione in caso di guasto del mini, basterà riportare il ritorno delle lampade e la fase di alimentazione al relè. Su questo aspetto mi sono già preparato perché ne ho qualcuno in più e sono tasmotati, ho già creato un backup per tutti.

**Reply by Supertux02** [reply]
@xyzmauro Si, io ho livinglight con i comandi classici e non hanno la luce integrata. Ho interpretato living smart come livinglight con comandi smart. Esteticamente sono molto simili...ovviamente le funzioni no. Per il momento mi sono fermato all'estetica (dato che la dolce metà guarda in primis l'estetica, poi la comodità "tattile"...tanto poi l'implementazione tocca a me -.-). 
Per gli shelly ho controllato...lo shelly 1 supporta i 16A, gli shelly 4pro plus supporta 16A x 4 canali...i 2.5 hai ragione, sono 10A. Quindi nel caso posso metterli negli interruttori, ma nelle prese devo pensarla diversamente.

**Reply by xyzmauro** [reply]
Guarda anche Vimar, non li ho mai provati ma sono Zigbee anche loro e se non sbaglio ci sono più linee estetiche

**Reply by Supertux02** [reply]
@Larry-78 Ciao! Sono tutt'orecchie...la soluzione mi interessa, ma sono un po' acerbo in fatto di impianti elettrici. In pratica tu faresti una sorte di mini quadro elettrico per stanza da dove comandi i punti luce e le prese elettriche, è corretto? Tu hai usato i sonoff mini con "andata e ritorno" dall'interruttore...io, usando gli shelly (facciamo l'1, per capirsi) mandando l'uscita alla presa SW potrei risolvere nello stesso modo...solo che invece che stare nella 503, starebbero nel mini quadro elettrico...ho capito?

**Reply by Supertux02** [reply]
@xyzmauro L'ho visto...ma 76€ per ogni presa non glieli darò mai :D

**Comment by essegi65** [comment]
si homekit controller giusto, scusate

**Reply by Maspar79** [reply]
CSupertux02 perchè dovresti mettere 503 aggiuntive per eventuali shelly? Chi ti obbliga ad inserire gli stessi nelle 503? Perchè non montarli nelle scatole di derivazione, a casa mia senza nessuna ristrutturazione (impianto elettrico di circa 30 anni) ho messo il 90% degli shelly dentro 3 scatole di derivazione.

**Reply by Supertux02** [reply]
Si, più o meno è la soluzione che mi ha suggerito Larry-78. Non mi dispiace come idea...alla fine mi permette di avere i vantaggi degli shelly senza la scomodità di occupare un posto nella 503. Mi resta un dubbio...se ho una luce che si accende da due interruttori diversi (con i deviatori), come devo collegarli? devo mettere X shelly secondo quanti sono gli interruttori oppure li faccio convergere tutti sulla presina SW dello shelly che poi è collegato al/ai punti luce?
Magari sono domande stupide, però nella vita mi occupo di tutt'altro e in questo periodo storico ho pochissimo tempo...per cui cerco di capirci qualcosa per quanto posso e l'idea di mettere tutto nelle 503 è, banalmente, perché non avevo pensato alle scatole di derivazione :-) Sicuramente chiederò aiuto ad un elettricista, ma vorrei sapere le cose in prima persona sia per capire cosa mi dice, sia per saper rispondere...anche perché so cosa vorrei e dovrei riuscire a spiegarlo in maniera corretta. 

Intanto colgo l'occasione per ringraziarvi tutti!!!

**Reply by Maspar79** [reply]
Se hai un punto luce che si accende da due o più punti si tratta di una deviata (da due punti con due deviatori, da 3 in su deviatori più invertitori) o pulsanti con relè passo passo. 
Con questo voglio dirti che in ogni caso basta un solo shelly, per esperienza personale posso consigliarti di optare per l'installazione di x pulsanti tanti quanti sono i punti da cui vuoi accendere e spegnere le tue luci, basta infatti collegare i pulsanti fra di loro in parallelo e farli confluire al tuo shelly. 
Affidati comunque ad un professionista per il rifacimento dell'impianto elettrico. Ciao

**Reply by Larry-78** [reply]
Quando i punti diventano piu di 2 si utilizzano gli invertitori  e si frappongono ai derivatori ma oramai sono da tempo sostituiti facilmente dai relè. Il consiglio che ho dato a @Supertux02 nasce dal fatto che ha intenzione di ristrutturare ed è meglio affidarsi a un elettricista per un progetto accurato. Nel mio caso ho provveduto al progetto personalmente perché sono un elettricista di professione e ho preferito installare cassette di derivazione più grandi per permettermi di installare comodamente i dispositivi. Non è  proprio un mini quadro elettrico come dici, è soltanto una cassetta dove convogliano tutti corrugati dei punti luci e prese. Poi all'interno vengono fatti i collegamenti delle varie linee dove poter accoppiare i dispositivi. Tutto parte dal quadro generale e percorre nel mio caso il corridoio che viene poi distribuito alle varie cassette, non necessariamente una per stanza, dipendente da come è distribuito il tuo appartamento. Ho preferito far viaggiare tutto a pavimento perché c'è stata la necessità di rifarlo. Per quanto riguarda il collegamento dello shelly ai deviatori il web è di aiuto, esistono una marea di immagini degli schermi

**Reply by sky7176** [reply]
Ciao,
io ho usato un sonoff 4chpro, che avendo un contatto in deviazione mi ha permesso di mantenere i vecchi interruttori e comunque di domotizzare le luci.
Certo devi fare qualche modifica all'impianto elettrico, **se non sai cosa fare non fare, con l'elettricità non si scherza si muore**, ma se sai come fare oppure lo fai fare da un esperto questo componente ti permette appunto di mantenere sia i comandi manuali sia quelli da domotica.

Giuseppe

**Reply by Supertux02** [reply]
@sky7176 Grazie per l'avvertimento, sono un medico...conosco i rischi collegati all'elettricità ;) . Ovviamente mi affiderò ad un elettricista per eseguire tutto, ma dato che poi la configurazione di home assistant la farò io voglio conoscere come sarà strutturato l'impianto per tanti motivi: per prima cosa, per la stima dei costi, poi per capire come mi conviene procedere per mettere in pratica ciò che ho chiaro in testa come risultato finale, poi come comportarmi con il geometra per tutta la questione normativa/bonus casa. Ci sono una miriade di cose da tenere in considerazione e non sarà una cosa che realizzerò "domani" ma voglio aver tutto chiaro su come procedere al fine anche di prevedere eventuali problemi e risolverli prima che si presentino (ovviamente nei limiti del possibile).
@Larry-78 Grazie, a questo punto penso che, per li interruttori, di far convergere tutti gli interruttori (messi come pulsanti) su un relé che poi si collega a dei sonoff4 pro flashati con tasmota, il quale poi esce diretto sui punti luce. Invece, per quanto riguarda le prese, le collegherei a degli shelly (da vedere se 1pm o 4pro...mi piacevano i 4pro plus ma sono introvabili) cosi da essere tranquillo sui carichi a 16A tipo aspirapolveri, phon e simili. Comunque come mini-quadro intendevo una sorta di centro di comando della stanza da cui poter controllare tutto...è improprio, non è un quadro elettrico nel senso stretto, lo so, ma penso si possa assimilare ad un mini centro di controllo prese-interruttori. E' corretto?
Grazie a tutti e scusate il ritardo nelle risposte!!

**Reply by Larry-78** [reply]
@Supertux02 si ti è chiaro il mio concetto, ma io per questioni di spazio e collegamento, visto che ho distribuito l'impianto su diverse linee, ho preferito i mini, utilizzare un sonoff4 è molto più ingombrante e tra l'altro per accoppiarlo a un relè dovresti saldare alcuni fili sulla scheda per poter attivare i relè integrati ad esso, il mini invece (anche gli shelly) hanno dei morsetti dedicati. Io ho utilizzato un sonoff4 per comandare due tapparelle e gli ho dedicato una cassetta installata accanto al cassone in alto. Ovviamente il mio è soltanto un punto di vista basato sulla mia esperienza, poi sei libero di progettare il tuo impianto come meglio credi sulla base di un parere di un esperto che potrebbe aiutarti nella realizzazione. Per quanto riguarda la questione burocratica, posso dirti che acquistare prodotti shelly dallo store italiano ti permette di usufruire di agevolazioni fiscali 4%, 10% oppure detrazioni fiscali al 50%, se dovessi avere i requisiti oggettivi che il sito richiede, potresti scaricarti il modulo. Per questioni di regolamento di questo forum non posso incollare qui il link ma basta cercare sul web. Se decidi di utilizzare anche sonoff invece dovresti richiedere la documentazione dal sito di acquisto, ovviamente non cinesi e co.

**Reply by Supertux02** [reply]
@Larry-78 Non sapevo delle agevolazioni...le ho trovate facilmente, grazie. Ora approfondisco la cosa. Nonostante sia shelly che sonoff siano ottimi prodotti, tra i due ho sempre preferito i primi, per cui se riuscissi ad avere "uno sconto" tramite agevolazioni sarei ben contento di optare per i primi. A questo punto valuterò anche l'idea di prendere una valanga di shelly 1 :D .
A questo punto faccio un ultima domanda "da elettricista" se mi permetti. Dato che stiamo pensando ad una ristrutturazione ad ampia scala, stavamo pensando anche ad una pompa di calore con conseguente impianto trifase. La mia idea sarebbe quella di mettere un contatore trifase ed a valle di questo mettere ciò che consuma di più (tipo la pompa di calore, eventualmente le piastre a induzione e via discorrendo). E' possibile a questo punto fare una sorta di "circuito sottoinsieme" in monofase a cui collegare tutto quello che ha consumi "normali" tipo luci, prese e tutto il resto (cosi da non impazzire con gli shelly)?

**Reply by Larry-78** [reply]
La risposta alla domanda è si, con una fornitura trifase si può distribuire l'impianto suddividendo le varie linee, per essere chiaro, tra il neutro e una singola fase c'è 220v e di conseguenza andrebbe equilibrato e distribuito tra le vaie fasi. Se non ho capito male l'eventuale pompa di calore sarà trifase? Di solito si preferisce una fornitura trifase se si ha intenzione di installare apparecchi trifase altrimenti basterebbe un semplice aumento di potenza monofase. In entrambi i casi andrebbero ovviamente fatti i calcoli giusti per determinare l'effettiva potenza da richiedere.

**Reply by Supertux02** [reply]
Diciamo che con l'aumento della potenza monofase arrivo al limite e, in ottica futura tra le piastre a induzione, i condizionatori, l'auto elettrica (quando sarà), il rischio di essere sottodimensionato ci sarebbe. Per cui volevo sapere se fosse tecnicamente possibile una soluzione in cui ciò che assorbe molto può andare in trifase ed il resto con la normale monofase (anche per la conseguente semplicità dell'impianto e dei costi conseguenti). Volevo sapere se si potesse fare cosi da fare un progetto da esporre all'elettricista, in caso contrario lo cambierei prima di proporlo facendo i cambiamenti del caso. Non c'è alcun dubbio che poi la realizzazione venga fatta dal professionista del caso ;)

**Reply by Larry-78** [reply]
ok adesso è più chiaro, mentre scrivevo non avevo considerato l'eventuale utilizzo di piastre, condizionatori etc. per potenze superiori a 6 kW si utilizza nella maggior parte dei casi la connessione trifase per evitare problemi di squilibrio sulla rete, ad ogni modo è possibile distribuire il tutto tra le varie fasi permettendoti di collegare anche apparecchiature monofase
